v2/cf-nodejs-logging-support/src/test/acceptance-test/child-logger.test.js

```diff
@@ -78,10 +78,6 @@ describe('Test child logger', function () {
             expect(childLogger.getTenantSubdomain).to.be.a('function');
         });
 
-        it.skip('implements setDynamicLoggingLevel()', function () {
-            expect(childLogger.setDynamicLoggingLevel).to.be.a('function');
-        });
```

Comment:
Is this functionality covered by a test somewhere else?

Reply:
Yes, it is covered in `global-context.test.js` under test `'Check log severity level'`